code/datums/elements/rad_insulation.dm

```diff
@@ -0,0 +1,34 @@
+/datum/element/rad_insulation
```

Comment:
[DMDOC compliance](https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md) would be welcome